cognite/client/data_classes/datapoints.py

```diff
@@ -116,6 +127,122 @@ class StatusCode(IntEnum):
     Bad = 0x80000000  # aka 1 << 31 aka 2147483648
 
 
+@dataclass(slots=True, frozen=True)
+class MaxOrMinDatapoint:
+    @abstractmethod
+    def dump(self, camel_case: bool = True) -> dict[str, Any]: ...
+
+    @classmethod
+    @abstractmethod
+    def load(cls, dct: dict[str, Any]) -> Self: ...
+
+
+@dataclass(slots=True, frozen=True)
+class MinDatapoint(MaxOrMinDatapoint):
+    timestamp: int
+    value: float
+
+    @property
+    def status_code(self) -> NoReturn:
+        raise AttributeError(
+            "'MinDatapoint' object has no attribute 'status_code'. Tip: fetch using `include_status=True`."
+        )
+
+    @property
+    def status_symbol(self) -> NoReturn:
+        raise AttributeError(
+            "'MinDatapoint' object has no attribute 'status_symbol'. Tip: fetch using `include_status=True`."
+        )
```

Comment:
Why these ones?

Reply:
Seems to me like an error that is very easy to make, so I have added very explicit user feedback